#Pick CXX and CXXFLAGS from EasyBuild via pre-defined variables
#May 11th 2016 by B. Hajgato (Free Uninrrsity Brussels, VUB)
--- kraken-0.10.5-beta/src/Makefile.orig	2015-02-18 12:38:00.000000000 +0100
+++ kraken-0.10.5-beta/src/Makefile	2016-05-11 13:33:39.370016057 +0200
@@ -1,5 +1,5 @@
-CXX = g++
-CXXFLAGS = -Wall -fopenmp -O3
+CXX ?= g++
+CXXFLAGS ?= -Wall -fopenmp -O3
 PROGS = db_sort set_lcas classify make_seqid_to_taxid_map db_shrink
 
 .PHONY: all install clean
